instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -250,7 +276,11 @@ def handler_init(init, handler, args, kwargs):
             "tornado.httpclient",
             "AsyncHTTPClient.fetch",
             partial(
-                fetch_async, tracer, client_request_hook, client_response_hook
+                fetch_async,
+                self.tracer,
+                client_request_hook,
+                client_response_hook,
+                self.response_size_histogram,
```

Comment:
You need another histogram to represent `http.client.response.size`

Reply:
Fixed